tools/meson-check-api-docs.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -eu
+
+LIBRARIES="${MESON_BUILD_ROOT}/libsystemd.so ${MESON_BUILD_ROOT}/src/libudev/libudev.so"
```

Comment:
It'd be nice to specify the list of libraries as arguments in meson, but this version works too.

Reply:
hmm, my meson-fu isn't yet advanced enough to make such things happen...

Reply:
ok, figured it out